src/pages/reports/index.astro

```diff
@@ -24,6 +24,6 @@ const items = reports.map(reportMapper);
 
 <Layout title="Reports">
   <PagesSection heading="Reports" color="gray-10">
-    <ContentItems items={items} />
```

Comment:
Since you switched to the `ReportList` component, should we delete the `ContentItems` component? Or do you want to leave that for later use?